### DISCUSSION TITLE
¿layout y rutas HTML dudas en el  section::after para que sirve?

### DISCUSSION THREAD

**Discussion by miguelapol** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#186] [Q&A]
tengo dudas para que funciona en el css saludos

**Comment by hackit92** [comment]
Es un pseudo elemento que te permite añadir al final elementos con la propiedad content, se utiliza generalmente de la mano con ::before

**Comment by doguedogue** [comment]
https://www.w3schools.com/cssref/sel_after.asp

https://developer.mozilla.org/es/docs/Web/CSS/::after
